reference/configuration.html.md

```diff
@@ -51,7 +51,7 @@ The optional build section contains key/values concerned with how the applicatio
 
 ```toml
 [build]
-  builder = "paketobuildpacks/builder:tiny"
+  builder = "paketobuildpacks/builder:base"
```

Comment:
This change was prompted by the following error:

```
Status: Downloaded newer image for gcr.io/paketo-buildpacks/nodejs:latest
Error validating stack mixins: buildpack paketo-buildpacks/node-engine@0.11.0 does not support stack io.paketo.stacks.tiny
```

I found that https://paketo.io/docs/#nodejs referenced the `base` tag and that resolved the error.